modules/p2/ding_list/ding_list.module

```diff
@@ -1164,3 +1158,20 @@ function ding_list_views_post_execute(&$view) {
     }
   }
 }
+
+function ding_list_user_overview_items_alter(&$rows, $account) {
```

Comment:
As I understand it `$rows` here are quite close to a render array. Could we add this using a Panels hook such as [`hook_panels_pane_content_alter`](http://api.drupalecommerce.org/api/panels/panels%21panels.api.php/7.x) instead of introducing a new hook?

Also: Please remember documentation.